src/episode.py

```diff
@@ -0,0 +1,36 @@
+from telegram.ext import Updater
+import requests
+from bs4 import BeautifulSoup
```

Comment:
We need to add this to the requirements.txt, otherwise the dependency will not be installed